src/schemas/v2/combined.json

```diff
@@ -36814,6 +36814,182 @@
               }
             }
           }
+        },
+        "valuationComparisonData": {
+          "type": "array",
+          "title": "Valuation Comparison Data",
+          "description": "A collection of data used to compare the valuations (market values) of properties in the real estate or property market. It typically includes various types of information about properties, such as recently sold properties, properties currently on sale, property pricing estimates, property characteristics, and more.",
+          "items": {
+            "type": "object",
+            "properties": {
+              "listingType": {
+                "title": "Listing Type",
+                "type": "string",
+                "enum": ["RecentlySold", "CurrentlyOnSale"]
+              },
+              "listingStatus": {
```

Comment:
what's the difference between `listingType` and `listingStatus`? Can a property be RecentlySold with status 'Sale'? or vice versa?

Reply:
The `listingType` and `listingStatus` properties serve different purposes in the schema:

1. `listingType`:
   - `listingType` represents the general category or type of property listing.
   - It has two possible values: "RecentlySold" and "CurrentlyOnSale."
   - "RecentlySold" indicates that the property listing represents a property that has recently been sold.
   - "CurrentlyOnSale" indicates that the property listing represents a property that is currently available for sale.

2. `listingStatus`:
   - `listingStatus` provides specific information about the status of a property listing, including whether it's currently for sale or has been sold.
   - It has two properties:
     - `type`: This property indicates whether the property is currently listed for "Sale" or has been "Sold."
     - `date`: This property represents the date and time of the listing status change, which can be used to track when a property transitioned from being on sale to being sold or vice versa.

To clarify, a property can have the following combinations of `listingType` and `listingStatus`:

- A property can have a `listingType` of "RecentlySold" and a `listingStatus` with `type` set to "Sold" to indicate that it has recently been sold.
- A property can have a `listingType` of "CurrentlyOnSale" and a `listingStatus` with `type` set to "Sale" to indicate that it is currently listed for sale.

The distinction between these properties allows for tracking both the type of listing (recently sold or currently on sale) and the specific status (sold or for sale) of a property, providing more detailed information about the property's history in the real estate market.
Now we have removed status to keep it simple